tests/codex/blockexchange/protobuf/testpresence.nim

```diff
@@ -26,12 +25,7 @@ checksuite "block presence protobuf messages":
     check message.price == @(price.toBytesBE)
 
   test "decodes CID":
-    check Presence.init(message).?cid == cid.some
-
-  test "fails to decode when CID is invalid":
```

Comment:
Why is this removed?

Reply:
Because we're no longer decoding from raw bytes to `Cid` inside of `Presence.init` proc. We're decoding it in [message.nim](https://github.com/codex-storage/nim-codex/pull/566/commits/8c1d97d75c97314c626b8544e17fe8b47d2d2613#diff-77bd53597755a1938432f3d6205bbfb4f9754f2ae7350f7faa8fc32f682c3b8cR161) (line 161)